parlai/agents/bart/bart.py

```diff
@@ -237,3 +237,12 @@ def _construct_token_losses(self, labels, model_output):
                 )
             )
         return token_losses
+
+
+class BartAgent_R3F(R3FMixin, BartAgent):
```

Comment:
Hm I think if this is `R3fAgent` then we can use `-m bart/r3f`

Reply:
`bart/bart/r3f` is apparently what works, but yeah, changing.

Reply:
I think we just gotta move it to bart/r3f.py

Reply:
I hate the import logic in parlai...